lib/puppet/provider/oneview_storage_pool/c7000.rb

```diff
@@ -43,6 +39,9 @@ def destroy
     super(:remove)
   end
 
+  # TODO: Swap this out for a simple message that the resource differs and should be dropped/recreated.
```

Comment:
Since there's already a lot of modifications in this PR, I'd rather not do it here.
I've opened the issue [144](https://github.com/HewlettPackard/oneview-puppet/issues/144), so that it gets fixed.